### DISCUSSION TITLE
Tilbakemelding i Portal

### DISCUSSION THREAD

**Discussion by Slaine2** [fremtind/jokul#2488] [Annet]
Det er ingen måte å gi tilbakemelding på de enkelte komponentene i vår portal. For å ha et best mulig Designsystem er tilbakemelding avgjørende.

For å øke trafikken i github og i DS-Forumet, tror jeg det er fornuftig å inkludere en tilbakemeldingsmodul. Slik blir det en lavterskel kontaktmulighet som tydeliggjør prosessen.

Hva synes dere?

**Comment by onordbo** [comment]
God ide! For at vi skal ha god effekt av en slik funksjon, synes jeg det er viktig at vi blir enige om hva slags mål en slik komponent skal ha. Jeg legger ved mine tanker nedenfor:

### ⚠️  Utfordinger
- Hvis man ikke er på Github eller Forum, så er det vanskelig å følge med i svingene på hva som rører seg av komponenter og div
- Det er knotete å huske på alle tingene man skal ha meg seg til forum i løpet av en uke, for man har kanskje ikke noen egen mekanikk for notere med seg alt man kommer på i løpet av en uke, frem til neste forum
- Frem til nå har vi ikke hatt noe felles sted å samle ting skriftlig og asynkront

### 💡  Forslag til målsetninger
- Vi ønsker oss mer deltakelse i Forum fra brukerne ute i leveranseteamene
- Vi ønsker å drive mer trafikk til Github Discussions
- Vi ønsker oss mer tilbakemelding på komponenter i portalen

### 🤔  Mine two-cents
Rent dokumentasjonsmessig, så vil jeg si at vi allerede har mye potensiale til å bruke Discussions for dette. Det er et funksjonsrikt område som lar oss sortere og kategorisere godt for fremtiden, for alt som handler om Jøkul, og vi slipper å rulle noe selv.

Når det er sagt, så kan jeg forstå verdien av å bygge inn en egen komponent i portalen for å gi tilbakemelding, da det gir brukerne av portalen veldig kort vei til å gi en rask feedback på noe de har sett eller lurer på, uten å måtte forville seg inn i discussions jungelen. 

Men da lurer jeg på, hva slags tilbakemelding ønsker vi oss der? Hva vil vi ha ut av den kommunikasjonen? Ønsker vi i det hele tatt direktefeedback via portalen? Eller burde en tilbakemeldings-komponent reklamere for Discussions området vårt, så vi får samlet alt på ett sted? Skal vi lage en felles mail-postkasse som kjerneteamet overvåker og legger ut innhold her, derifra?

***

# TL;DR: Det er lettere for portalbrukere å skrive en kjapp kommentar i et tekst-inputfelt i portalen, enn å lage en hel discussions-greie, men det er uklart for meg hva slags verdi vi ønsker oss ut av en slik mekanisme direkte i portalen.

**Comment by piofinn** [comment]
Jeg er veldig enig i at vi bør ha en feedback-modul i designsystemportalen, men jeg tror _ikke_ jeg er enig i at den bør være der for å gi oss tilbakemelding på selve _innholdet_ i designsystemet.

For meg virker formålet med en feedback-modul ganske rett frem: Skaffe bedre innsikt i hvordan **portalen** er å bruke. Det er dette vi bruker Feedback-komponenten til i løsningene vi lager for kunder/interne, så den burde forhåpentligvis gjøre samme nytte for oss 😅 I mine øyne bør det ikke være noe (eller veldig lite) overlapp mellom innsikten herfra og fra discussions/forum/utvidet standup.

Dersom vi ønsker kortere vei til å diskutere **innholdet** i designsystemet bør vi IMO heller lage kortere vei til der vi diskuterer disse tingene, altså GitHub Discussions, forum og utvidet standup. Vi har allerede lenker til koden på GitHub øverst på alle komponentsidene, hva med å legge til en lenke til Discussions også? Hvis Discussions er en jungel, som @onordbo sier, må vi kanskje heller se på hvordan vi bedre kan strukturere diskusjonene der?

**Reply by piofinn** [reply]
Jeg tror heller ikke det er feil å ha en egen side i portalen som handler om å diskutere innholdet i designsystemet; en slags hub for diskusjon som refererer til dokumentasjon, forum, Discussions, etc. Noe av det behovet kommer kanskje til å bli dekket av den nye "kom i gang"-siden som Øyvind holder på med?

**Reply by onordbo** [reply]
Absolutt @piofinn. Jeg tror vi kommer langt med litt oppdaterte "kom i gang" sider både når det gjelder bidrag og designer-onboarding. Når det er sagt så eksperimenterer vi med en tilbakemeldingsmodul i portalen, dog ser vi for øyeblikket mest på muligheten for å promotere og oppfordre til mer engasjement i diskusjoner på GitHub enn å sende oss skriftlige direktetilbakemeldinger via portalen. Hvis man ser feil eller endringer man ønsker i innholdet, burde man tross alt nå kunne være utstyrt til å gjøre den endringen selv, eller alternativt starte en ny tråd på det 😄

**Reply by piofinn** [reply]
Hvis målet er å oppfordre til engasjement og diskusjoner på GitHub er det vel strengt tatt ikke snakk om tilbakemelding 😅 Er kanskje derfor jeg slet litt med å forstå diskusjonen her.

Men i lys av alt vi har snakket om de siste månedene (ref. ~~prosess~~ kultur o.l.) tenker jeg at det ville vært veldig nyttig med en faktisk tilbakemeldingsmodul også. Noe så enkelt som "Fant du det du lette etter"-preseten i Feedback? Når vi uansett vil utvide og forbedre portalen er det jo fint med litt faktisk tilbakemelding fra brukerne om hva de ønsker å se 😊

**Comment by Slaine2** [comment]
Et annet spørsmål er om det kun skal være mulig å ta kontakt internt (github / teams) eller er det av interesse å tilby en kontaktmulighet til eksterne? Og er tilbakemeldingsmodulen det rette instrumentet for dette? Hva mener dere alle sammen?

Et godt eksempel for meg, hvor muligheten også hadde vært spennende for utenforstående, var etter å ha vunnet designprisen. Jeg tror vi absolutt hadde mye trafikk på vår side, og noe som absolutt ville gitt tilbakemeldinger, som hadde vært spennende for oss, hvordan Jøkul oppfattes.
